rtl/2-stage-1024-cache.v_toplevel_RegFilePlugin_regFile.bin

```diff
@@ -0,0 +1,32 @@
+00000000000000000000000000000000
```

Comment:
This CL shouldn't be adding an `rtl` directory. All `rtl` is under the `gateware` directory.

Reply:
I've moved files from `rtl` directory to `gateware`

Reply:
I've moved files from rtl directory to gateware